src/cliframe/components/TriCommandLine.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react"
+import * as Redux from "redux"
+import Logger from "@src/lib/logging"
+
+const logger = new Logger("cmdline")
+
+interface TriCommandLineProps {
+    store: Redux.Store<any, Redux.AnyAction>
+}
+
+export default function TriCommandLine({
+    store,
+}: TriCommandLineProps): JSX.Element {
+    return <div id="tridactyl-commandline">
+        <span id="tridactyl-colon"></span>
+        <input id="tridactyl-input"
+               ref={this.inputRef}
```

Comment:
something, somewhere, doesn't like this this.

It seems to come from `cliframe/index.tsx:10`.